api/OffscreenCanvas.json

```diff
@@ -214,7 +217,7 @@
             "webview_android": "mirror"
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
```

Comment:
@queengooborg I'm getting an error for lines below

```
api.OffscreenCanvas.convertToBlob - Error → Basic support in firefox was declared implemented in a later version (105) than the following sub-feature(s):
        → api.OffscreenCanvas.convertToBlob.option_type_parameter_webp: 96
```

Presumably the solution is just to update support from 96 to 105 (this should have been behind the pref too "in theory")

Reply:
Actually, I'll just update it to 105. That makes sense given that's when this release is "public"